lib/transactions.js

```diff
@@ -99,6 +108,25 @@ TxController.prototype.transformTransaction = function(transaction, options, cal
 
   transformed.txlock = transaction.txlock;
 
+  if (transaction.proRegTx !== undefined) {
+    transformed.preRegTx = transaction.preRegTx;
```

Comment:
s/pre/pro/g ?

Reply:
Yepp, this was a type :see_no_evil: fixed and pushed